src/services/legal-services.ts

```diff
@@ -3,6 +3,7 @@ import { StatusCodes } from 'http-status-codes'
 import { AmalgamationFilingIF, BusinessIF, DissolutionFilingIF, IncorporationFilingIF, NameRequestIF,
   RegistrationFilingIF, RestorationFilingIF } from '@/interfaces'
 import { FilingTypes } from '@/enums'
+import { BusinessLookupServices } from '@/services'
```

Comment:
I'd prefer to not have this dependency.

I recommend that you copy the accountId getter here.

Reply:
will do